kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -325,7 +325,7 @@ define([
                     break;
 
                 case 'new_job':
-                    Jupyter.notebook.save_checkpoint();
+                    Jupyter.narrative.saveNarrative();
```

Comment:
This little wrapper should be called to ensure Narrative-specific things get handled during the save. Such as preventing looking up the document version.